Source/WebCore/layout/formattingContexts/inline/text/TextUtil.cpp

```diff
@@ -372,6 +373,17 @@ LineBreakIteratorMode TextUtil::lineBreakIteratorMode(LineBreak lineBreak)
     return LineBreakIteratorMode::Default;
 }
 
+TextBreakIterator::ContentAnalysis TextUtil::contentAnalysis(const WordBoundaryDetection& wordBoundaryDetection)
+{
+    return WTF::switchOn(wordBoundaryDetection, [](WordBoundaryDetectionNormal) {
+        return TextBreakIterator::ContentAnalysis::Mechanical;
+    }, [](WordBoundaryDetectionManual) {
+        return TextBreakIterator::ContentAnalysis::Mechanical;
+    }, [](const WordBoundaryDetectionAuto&) {
+        return TextBreakIterator::ContentAnalysis::Linguistic;
```

Comment:
Do we need to do special handling for the specified language eventually? If not, do you think the CSS syntax should be changed to remove the language string?

Reply:
https://github.com/w3c/csswg-drafts/issues/7193 is about removing the specified language.